pages/products.html

```diff
@@ -1 +1,328 @@
+<!DOCTYPE html>
+<html lang="en">
 
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://altur-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="../style.css">
+
+    <link rel="stylesheet" href="../stylesheets/products.css">
```

Comment:
just a small suggestion like you can import your CL link the base CSS file and import only one base or main CSS file into all your HTML header files will easier for you only have link to add in Html

Reply:
Ok, I'll do it. Thanks for Pointing out.